### PR TITLE
Tabs component

### PR DESCRIPTION
**Pull request by Dietr**
Adds a tab component according to the webuniversum design:
Screenshot 2022-05-16 at 14 50 09

### CODE REVIEW

File: addon/components/au-tabs.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<nav class="au-c-tabs {{if @reversed 'au-c-tabs--reversed'}}" ...attributes>
+  <ul class="au-c-tabs-list">
+    {{yield}}
```
Comment (Windvis) [inline addon/components/au-tabs.hbs:3]: Impromptu review, but I would yield the component here as well, similar to the  recent AuToolbar PR. I don't think there is a use-case for using the tabs-item component directly?

**Comment by Windvis** [comment]
@Dietr FYI, "enhancements" are new features so they should be released as minor versions according to the SemVer spec 😄.

File: addon/components/au-tabs.hbs (hunk 1 above)
Comment (Dietr) [reply addon/components/au-tabs.hbs:3]: Ok, will fix.

**Comment by Dietr** [comment]
Sure, will follow this logic for future releases.

**Pull request merged**